setup.py

```diff
@@ -135,6 +135,10 @@ def find_packages_by_root_package(where):
         'formatters': [
             "behave-html-formatter",
         ],
+        'toml': [ # Enable pyproject.toml support.
+            "tomli>=1.1.0; python_version >=  '3.0' and python_full_version < '3.11.0a7'",
+            "toml>=0.10.2; python_version <  '3.0'", # py27 support
```

Comment:
Flake8 wants us to place 2 spaces before comments.

What if we have Python >= 3.11.0a7? Neither tomli nor toml will be installed, will they? Do we have a solution for the user?